sdk/go/common/workspace/plugins.go

```diff
@@ -422,7 +422,7 @@ func (source *githubSource) getHTTPResponse(
 		addAuth = " You can set GITHUB_TOKEN to make an authenticated request with a higher rate limit."
 	}
 
-	logging.Errorf("GitHub rate limit exceeded%s%s", tryAgain, addAuth)
+	logging.Errorf("GitHub rate limit exceeded for host%s%s%s", req.URL.Host, tryAgain, addAuth)
```

Comment:
Why just the host? Be more useful to print the whole URL here wouldn't it?

Reply:
I think this is the full url, but happy to reassess?

Reply:
https://pkg.go.dev/net/url#URL
> Host        [string](https://pkg.go.dev/builtin#string)    // host or host:port

I think we just want `req.URL` here to just get the result of `String()` here which will include the path.

Reply:
@Frassle fixed!